packages/utils/src/index.ts

```diff
@@ -241,3 +241,20 @@ export const versionGreaterOrEqualTo = (
  * @returns The new string
  */
 export const upcase = (str: string) => str.substr(0, 1).toUpperCase() + str.substr(1);
+
+/**
+ * Turns a passed in string into
+ * a human readable label
+ * @param {String} str The string to convert.
+ * @returns The new string
+ */
+export const humanize = (str: string) => {
+  let label = str
+    .replace(/([a-z])([A-Z]+)/g, '$1 $2')
+    .split(/\s|_|\-/)
+    .filter(i => i)
+    .map(upcase)
+    .join(' ');
+
+  return label;
+};
```

Comment:
Any reason not to do this as a simple return value?
```suggestion
export const humanize = (str: string) => str
  .replace(/([a-z])([A-Z]+)/g, '$1 $2')
  .split(/\s|_|\-/)
  .filter(i => i)
  .map(upcase)
  .join(' ');
```